sympy/parsing/tests/test_autolev.py

```diff
@@ -90,6 +109,10 @@ def test_dynamics_online():
                 in_filepath = os.path.join("dynamics-online", name, i + ".al")
                 out_filepath = os.path.join("dynamics-online", name, i + ".py")
                 _test_examples(in_filepath, out_filepath, i)
+            for i in ch:
+                in_filepath = os.path.join("dynamics-online", name, i + ".al")
+                out_filepath = os.path.join("dynamics-online", name, i + ".py")
+                _test_examples(in_filepath, out_filepath, i, include_numpy=True)
```

Comment:
It isn't clear to me that both the True and False versions of this flag are tested.

Reply:
The default argument is a flag that was used to switch on/off of numpy/math modules import